Source: written by Calvin Walton <calvin.walton@kepstin.ca>
Upstream: contacted via email form, response pending.
Reason: Makefile attempts to install binary to /usr/sbin without creating the
 directory first.

diff -u atop-2.1-orig/Makefile atop-2.1/Makefile
--- atop-2.1-orig/Makefile	2014-09-06 09:12:52.000000000 -0400
+++ atop-2.1/Makefile	2015-02-21 12:47:30.271447263 -0500
@@ -49,6 +49,8 @@
 			then mkdir -p $(DESTDIR)$(LOGPATH); fi
 		if [ ! -d $(DESTDIR)$(BINPATH) ]; 	\
 			then mkdir -p $(DESTDIR)$(BINPATH); fi
+		if [ ! -d $(DESTDIR)$(SBINPATH) ]; 	\
+			then mkdir -p $(DESTDIR)$(SBINPATH); fi
 		if [ ! -d $(DESTDIR)$(SCRPATH) ]; 	\
 			then mkdir -p $(DESTDIR)$(SCRPATH); fi
 		if [ ! -d $(DESTDIR)$(MAN1PATH) ]; 	\
